engine/org.eclipse.birt.report.engine/src/org/eclipse/birt/report/engine/script/internal/ExtendedItemScriptExecutor.java

```diff
@@ -40,27 +40,41 @@ public static void handleOnPrepare(ExtendedItemHandle handle, ExecutionContext c
 
 	public static void handleOnCreate(ExtendedItemDesign design, IContent content, ExecutionContext context) {
 		ExtendedItemHandle handle = (ExtendedItemHandle) design.getHandle();
```

Comment:
Move this item after the next if block. No need to get the handle if it is not used.